### ISSUE TITLE
[doc] change from "new DualHRangeBar()" to "new DualHRange()" in README 

### ISSUE THREAD

**Issue by penghuili** [celestialphineas/dual-range-bar#16]
Thanks for this nice little lib!

Readme seems using the wrong class to create the instance.
For example here: https://github.com/welai/dual-range-bar/tree/v1.0.3?tab=readme-ov-file#configuring

It should be `new DualHRange()`, not `new DualHRangeBar()`.

At least if I use the version from unpkg.com like this
```
<script src="https://unpkg.com/dual-range-bar@1.0.3/dist/dual-range-bar.min.js"></script>
```

I get this:
![Screenshot 2024-06-01 130725](https://github.com/welai/dual-range-bar/assets/5326739/6115f58f-ed9e-4a08-8ae4-a713f5cbc564)

Maybe I miss something?

BTW, I am using it in remiind.cc :)

**Comment by celestialphineas** [comment]
Thank you for pointing out! This has been fixed in the latest release [v1.1.0](https://github.com/welai/dual-range-bar/releases/tag/v1.1.0). Now the exported names to the `window` object are consistent with other exporting paradigms, and follow the documented behavior as well.

**Comment by penghuili** [comment]
thank you for the quick fix!
